app/routes.js

```diff
@@ -29,7 +29,8 @@ export const routes = (authProvider) => {
   router.get('/oauth/callback', authProvider.loginCallback());
 
   router.get('/logout', async (req, res) => {
-    const url = await authProvider.logout();
+    const idToken = req.session && req.session.accessToken && req.session.accessToken.id_token;
```

Comment:
use that extractAccessToken function here?

Reply:
that is the id token but could do something

Reply:
it's identical ain't it? or am i missing something?

Reply:
`id_token` and `access_token` at the end but I had the helper wrong so changed it now

Reply:
don't fancy adding a little helper function for this too. The other one made me happy.

Reply:
I have

Reply:
yeah just seen it. Sorry it hadn't updated